ERROR: 0:36: 'switch' : Reserved word.
ERROR: 0:36: 'switch' : syntax error: syntax error
--- main.py.orig	2014-11-02 20:30:44.000000000 -0600
+++ main.py	2022-01-12 15:43:12.000000000 -0600
@@ -30,7 +30,7 @@
     shader_module.ShaderFragment("""
 //#define BLACK_AND_WHITE
 #ifndef BLACK_AND_WHITE
-    #define HAS_SWITCH
+//    #define HAS_SWITCH
     #define HAS_MOD
     #ifdef HAS_MOD
         #extension GL_EXT_gpu_shader4 : enable /*for "%"*/
